package.json

```diff
@@ -349,6 +349,10 @@
       "prettier --write",
       "node ./bin/verify-doc-links"
     ],
+    "e2e/**/*.{js,jsx,ts,jsx}": [
```

Comment:
Since https://github.com/metabase/metabase/pull/28749, we haven't add a step to run ESLint and prettier on files under `e2e/`

Reply:
🏆